### ISSUE TITLE
minikube node add --- E0921 19:36:38.987203   11003 register.go:123] unexpected first step: ""

### ISSUE THREAD

**Issue by harryttd** [kubernetes/minikube#9295]
**Steps to reproduce the issue:** 

1. `minikube start`
2. `minikube node add`

**Full output of ~failed~ command:** 

```
❯ minikube node add
😄  Adding node m02 to cluster minikube
❗  Multi-node clusters are currently experimental and might exhibit unintended behavior.
📘  To track progress on multi-node clusters, see https://github.com/kubernetes/minikube/issues/7538.
E0921 20:36:24.655586   14298 register.go:123] unexpected first step: ""
👍  Starting node minikube-m02 in cluster minikube
E0921 20:36:24.678625   14298 register.go:123] unexpected first step: ""
🔥  Creating hyperkit VM (CPUs=2, Memory=2200MB, Disk=20000MB) ...
E0921 20:36:40.854030   14298 register.go:123] unexpected first step: ""
🐳  Preparing Kubernetes v1.19.2 on Docker 19.03.12 ...
E0921 20:36:48.744292   14298 register.go:123] unexpected first step: ""
🔎  Verifying Kubernetes components...
🏄  Successfully added m02 to minikube!
```
**Full output of `minikube start` command used, if not already included:**
```
😄  minikube v1.13.1 on Darwin 10.15.6
✨  Automatically selected the hyperkit driver
💾  Downloading driver docker-machine-driver-hyperkit:
    > docker-machine-driver-hyperkit.sha256: 65 B / 65 B [---] 100.00% ? p/s 0s
    > docker-machine-driver-hyperkit: 10.90 MiB / 10.90 MiB  100.00% 6.40 MiB p
🔑  The 'hyperkit' driver requires elevated permissions. The following commands will be executed:

    $ sudo chown root:wheel /Users/foo/.minikube/bin/docker-machine-driver-hyperkit
    $ sudo chmod u+s /Users/foo/.minikube/bin/docker-machine-driver-hyperkit


Password:
💿  Downloading VM boot image ...
    > minikube-v1.13.1.iso.sha256: 65 B / 65 B [-------------] 100.00% ? p/s 0s
    > minikube-v1.13.1.iso: 173.91 MiB / 173.91 MiB [] 100.00% 8.74 MiB p/s 20s
👍  Starting control plane node minikube in cluster minikube
💾  Downloading Kubernetes v1.19.2 preload ...
    > preloaded-images-k8s-v6-v1.19.2-docker-overlay2-amd64.tar.lz4: 486.36 MiB
🔥  Creating hyperkit VM (CPUs=2, Memory=6000MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.19.2 on Docker 19.03.12 ...
🔎  Verifying Kubernetes components...
🌟  Enabled addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube" by default
```

**Comment by medyagh** [comment]
@harryttd thanks for creating this issue, this seems to be a noise from the logging from multinode, we should fix that. it doesn't appear to affect the functionality.

**Comment by chirangaalwis** [comment]
Experiencing the same issue with

```
minikube version: v1.16.0
commit: 9f1e482427589ff8451c4723b6ba53bb9742fbb1
```

**Comment by RogueCultivators** [comment]
![image](https://user-images.githubusercontent.com/49176713/108790118-e3b94680-75b6-11eb-8dbc-24f564b73e50.png)

**Comment by gkgaurav31** [comment]
I got the same error when adding another node. 

➜  services git:(master) ✗ minikube node add
😄  Adding node m02 to cluster minikube
❗  Cluster was created without any CNI, adding a node to it might cause broken networking.
E0729 17:26:10.027486  564491 register.go:141] unexpected first step: ""
👍  Starting node minikube-m02 in cluster minikube
E0729 17:26:10.041995  564491 register.go:141] unexpected first step: ""
🚜  Pulling base image ...
E0729 17:26:10.102235  564491 register.go:141] unexpected first step: ""
🔥  Creating docker container (CPUs=2, Memory=2200MB) ...\ E0729 17:26:49.521952  564491 register.go:141] unexpected first step: ""

🐳  Preparing Kubernetes v1.20.7 on Docker 20.10.7 ...| E0729 17:26:57.624998  564491 register.go:141] unexpected first step: ""

🔎  Verifying Kubernetes components...
🏄  Successfully added m02 to minikube!

I also found another issue (which could be related to the above issue).

My Pod and Service YAML files are as below:

**service.yml:**

```
apiVersion: v1
kind: Service
metadata:
  name: hello-svc
spec:
  type: NodePort
  selector:
    name: hello-world-pod
  ports:
  - port: 8080
    targetPort: 8080
```

**pod.yml:**

```
apiVersion: v1
kind: Pod
metadata:
  name: hello-world-pod
  labels:
    name: hello-world-pod
    app: web
spec:
  containers:
  - name: hello-world-pod
    image: gkgaurav31/node-app
    resources:
      limits:
        memory: "128Mi"
        cpu: "500m"
    ports:
      - containerPort: 8080
```

It works if I start the cluster with the command:
**minikube start --nodes 2** 

But, the service does not work if I start the cluster using **minikube start** and then add another node using the command minikube **node add**. 

![image](https://user-images.githubusercontent.com/9574170/127538795-2098d32d-eceb-4192-8e99-09939203443e.png)
